packages/node_modules/@webex/plugin-meetings/src/meeting-info/index.js

```diff
@@ -108,17 +108,16 @@ export default class MeetingInfo extends StatelessWebexPlugin {
    * @memberof MeetingInfo
    */
   fetchMeetingInfo(destination, type = null) {
-    const options = this.fetchInfoOptions(destination, type);
-
+    return this.fetchInfoOptions(destination, type).then((options) =>
     // fetch meeting info
-    return this.requestFetchInfo(options).catch((error) => {
+      this.requestFetchInfo(options).catch((error) => {
```

Comment:
Need to keep the promise chain:
```suggestion
      return this.requestFetchInfo(options).catch((error) => {
```

Reply:
will do

Reply:
it auto returns with arrow syntax, can you double check? it shows locally as

```
    return this.fetchInfoOptions(destination, type).then((options) =>
    // fetch meeting info
      this.requestFetchInfo(options).catch((error) => {
```

Reply:
Yes, apologies, I guess I missed that from the previous line.